app/main.py

```diff
@@ -1 +1,27 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    try:
+        parts = command.split()
+        if len(parts) != 3 or parts[0] != "mv":
+            raise ValueError("Invalid command format. Expected: mv")
```

Comment:
The error message for the `ValueError` should be more informative. Consider specifying the expected command format, such as 'Expected format: mv <source> <destination>'.